resources/js/Pages/Entry/Create/DmEntryCreate.tsx

```diff
@@ -0,0 +1,24 @@
+import {usePage} from '@inertiajs/inertia-react'
+import {ThemeProvider} from '@mui/material/styles'
+import {DmEntryCreateForm} from 'Components'
+import {ApplicationLayout} from 'Layouts'
+import React from 'react'
+import {UsePageType} from 'Types/global'
+import {getFontTheme} from 'Utils'
+
+const DmEntryCreate = () => {
+    const theme = getFontTheme('Form', 16)
+    const {auth, adventures} = usePage<UsePageType>().props
```

Comment:
don't use this, get it from props

Reply:
how? the component isn't accessible from the ApplicationLayout

Reply:
`const DmEntryCreate = ({adventures, auth}) => `

will get you what you want

Reply:
obviously you need to type them